website/docs/exempt-namespaces.md

```diff
@@ -65,6 +67,10 @@ If it becomes necessary to exempt a namespace from Gatekeeper webhook entirely (
    3. Add the `admission.gatekeeper.sh/ignore` label to the namespace. The value attached
       to the label is ignored, so it can be used to annotate the reason for the exemption.
 
+Similarly, you can also exempt entire groups of namespaces using the `--exempt-namespace-prefix` and `--exempt-namespace-suffix` flags.
```

Comment:
was there a need to update the helm chart readme too? as per 

https://github.com/open-policy-agent/gatekeeper/pull/2655/files

Reply:
@salaxander Can you please address this as well? Thanks!

Reply:
So it looks like the `exemptNamespaceSuffixes` Helm value doesn't exist (I would consider adding a new value to be out of scope for this PR). `exemptNamespacePrefixes` is documented on line 176 of the README

```
| controllerManager.exemptNamespacePrefixes     | The namespace prefixes to exempt by the admission webhook  
```

Reply:
Helm chart values changes shouldn't be that big. You can take a look at https://github.com/open-policy-agent/gatekeeper/pull/2655/files but address the review comment. Or you can do it as a follow up.

Reply:
i am okay if we want to follow up with that if we can create an issue